Dockerfile

```diff
@@ -19,28 +24,21 @@ RUN mkdir /projects ${HOME} && \
       chmod -R g+rwX ${f}; \
     done
 
-# odo and oc versions have to match the ones defined in https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/tools.json
-ENV GLIBC_VERSION=2.30-r0 \
-    ODO_VERSION=v1.0.1 \
-    OC_VERSION=v3.11.0 \
-    OC_TAG=0cbc58b \
-    KUBECTL_VERSION=v1.16.2
-
 # plugin executes the commands relying on Bash
 RUN apk add --update --no-cache bash && \
     # install glibc compatibility layer package for Alpine Linux
     # see https://github.com/openshift/origin/issues/18942 for the details
     wget -O glibc-${GLIBC_VERSION}.apk https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk && \
     apk --update --allow-untrusted add glibc-${GLIBC_VERSION}.apk && \
     rm -f glibc-${GLIBC_VERSION}.apk && \
-    # install oc
-    wget -O- https://github.com/openshift/origin/releases/download/${OC_VERSION}/openshift-origin-client-tools-${OC_VERSION}-${OC_TAG}-linux-64bit.tar.gz | tar xvz -C /usr/local/bin --strip 1 && \
-    # install odo
+    # get oc
+    wget -O - https://mirror.openshift.com/pub/openshift-v3/clients/${OC_VERSION}/linux/oc.tar.gz | tar -xz oc -C /usr/local/bin && \ 
```

Comment:
do we need this change ?

Reply:
that depends. do you NEED CVE fixes? I mean there's a reason why oc 3.11 is on its 170th re-release. Is there a reason why the upstream vsix team is still using 3.11.0 instead of 3.11.170? Are they afraid of bugfixes? :D

Reply:
I mean why do we need changes if we bump OC_VERSION only ( not URL)

Reply:
OC 3.11.0 can be downloaded from here: https://github.com/openshift/origin/releases/download/${OC_VERSION%%-*}/openshift-origin-client-tools-${OC_VERSION}-linux-64bit.tar.gz

But all the other 3.11.x versions are NOT in github releases. Instead they are here:

https://mirror.openshift.com/pub/openshift-v3/clients/${OC_VERSION}/linux/oc.tar.gz

They are also packaged differently, which is why the untar command is different.